imports/redux/modules/editor.js

```diff
@@ -0,0 +1,96 @@
+import { makeEditorModel } from '../../util/editor-model-factory';
+
+const SET_LOCAL_EDITOR_CONTENT = 'SET_LOCAL_EDITOR_CONTENT';
+const SET_STREAM_EDITOR_CONTENT = 'SET_STREAM_EDITOR_CONTENT';
+const SET_STREAM_EDITOR_LEADER = 'UPDATE_STREAM_EDITOR_LEADER';
+const UPDATE_STREAM_EDITOR_USERS = 'UPDATE_STREAM_EDITOR_USERS';
+const UPDATE_AVAILABLE_STREAMS_LIST = 'UPDATE_AVAILABLE_STREAMS_LIST';
+const INIT_STREAM_EDITOR_MODEL = 'INIT_STREAM_EDITOR_MODEL';
+const INIT_LOCAL_EDITOR_MODEL = 'INIT_LOCAL_EDITOR_MODEL';
+const CHANGE_CURRENT_STREAM = 'CHANGE_CURRENT_STREAM';
+
+export default function reducer(state = {
+  localEditor: {
+    editorModel: undefined,
+    filePath: 'untitled.txt',
+    viewState: {},
+  },
+  streamEditor: {
+    _id: undefined,
+    editorModel: undefined,
+    viewState: {},
+    leader: undefined,
+    owner: undefined,
+    users: [],
+    focused: false,
+    active: false,
+    showLeaderMessage: false,
+  },
```

Comment:
Initializing all these properties to `undefined` is weird. I'm guessing it's a leftover from Typescript?